recipe/meta.yaml

```diff
@@ -32,14 +32,12 @@ outputs:
         - patch  # [unix]
       host:
         - python
-        - pip >=9.0.1
+        - pip
         - certifi >=2020.06.20
         - freetype 2.10
-        - numpy {{ numpy }}
```

Comment:
how come? upstream is `>=1.21,<2`

Reply:
Good catch, I took 1.25 from pyproject.toml. But that is because they changed from oldest-supported-numpy as numpy 1.25 also support older version. I will correct.

Reply:
I'm personally not sure where they saw that starting from 1.25, oldest-supported-numpy isn't necessary anymore. From what I know, this will become possible in 2.0.... not 1.25...

Reply:
hmm well I don't know either